### PR TITLE
Update Cloud Run Samples to use Jib

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

**Comment by averikitsch** [comment]
@kurtisvg can you PTAL on why kokoro isn't passing?

**Comment by averikitsch** [comment]
Solution: run `mvn clean` during clean up so Kokoro doesn't try to upload the cached files.

File: run/README.md [hunk 1]
```diff
@@ -9,71 +9,73 @@ This directory contains samples for [Google Cloud Run](https://cloud.run). [Clou
...
-
-    PORT=8080 && docker run --rm \
-        -p 8080:${PORT} \
-        -e PORT=${PORT} \
-        -e GOOGLE_APPLICATION_CREDENTIALS=/tmp/keys/${SA_KEY_NAME}.json \
-        -v $GOOGLE_APPLICATION_CREDENTIALS:/tmp/keys/${SA_KEY_NAME}.json:ro \
-        -v $PWD:/app $SAMPLE
+    ```Bash
+    PORT=8080 && docker run \
+       -p 9090:${PORT} \
```
Comment (kurtisvg) [inline run/README.md]: nit: Is this supposed to be 9090?

File: run/hello-broken/pom.xml [hunk 2]
```diff
@@ -70,24 +70,18 @@ limitations under the License.
...
-              <mainClass>com.example.cloudrun.App</mainClass>
-            </manifest>
-          </archive>
-          <descriptorRefs>
-            <descriptorRef>jar-with-dependencies</descriptorRef>
-          </descriptorRefs>
+          <to>
+            <image>gcr.io/PROJECT_ID/hello-broken</image>
+          </to>
+    <!-- <container>
```
Comment (kurtisvg) [inline run/hello-broken/pom.xml]: What is this comment for? Can we either add more context, or remove it?

File: run/README.md (hunk 1 above)
Comment (averikitsch) [reply run/README.md]: Yes, based on https://cloud.google.com/run/docs/testing/local. I've added a note to clarify

File: run/hello-broken/pom.xml (hunk 2 above)
Comment (averikitsch) [reply run/hello-broken/pom.xml]: comment added.

**Pull request merged**